src/commonMain/kotlin/fr/acinq/lightning/channel/states/WaitForOpenChannel.kt

```diff
@@ -24,7 +24,7 @@ data class WaitForOpenChannel(
     val temporaryChannelId: ByteVector32,
     val fundingAmount: Satoshi,
     val pushAmount: MilliSatoshi,
-    val wallet: WalletState,
+    val utxos: List<WalletState.Utxo>,
```

Comment:
I'm nitpicking but it would really be clearer if those were named `walletInputs` in all commands and channel states

```suggestion
    val walletInputs: List<WalletState.Utxo>,
```